src/components/OurStoriesArticles/style.scss

```diff
@@ -0,0 +1,47 @@
+.our-stories{
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 75px;
+
+
+  &__item{
+
+    &:nth-child(1), &:nth-child(2), &:nth-child(3){
+      display: flex;
+      width: 370px;
+      font-family: Seconda;
```

Comment:
if you write inside  `&:nth-child(1), &:nth-child(2), &:nth-child(3){`
something like this `& .our-stories__title` it's don't work?

Reply:
`.our-stories__item-title`
`.our-stories__item-img`
`.our-stories__item-info`
`.our-stories__item-date`